### ISSUE TITLE
Question about training DeiT-small distilled

### ISSUE THREAD

**Issue by mingqiJ** [facebookresearch/deit#186]
Hi,

I want to train DeiT-small distilled now.

Is this command okay?
`python -m torch.distributed.launch --nproc_per_node=4 --use_env main.py --model deit_small_patch16_224 --batch-size 256 ----distillation-type hard --data-path /path/to/imagenet --output_dir /path/to/save`

Thanks

**Comment by mingqiJ** [comment]
When I use this command to train, it still take errors
`python3 -m torch.distributed.launch --nproc_per_node=4 --use_env main.py --model deit_small_patch16_224 --batch-size 256 --seed 0 --teacher-model regnety_160 --distillation-type hard --teacher-path /scratch/regnety_160-a5fe301d.pth --data-path /scratch/pytorch-image-models-master/imagenet --output_dir /scratch/deit-main/distill_output`
<img width="539" alt="WeChat125a9865a1e83a1d3b179f8689e71b7f" src="https://user-images.githubusercontent.com/98499594/189306454-c320d2ac-e172-4c76-b03b-4317891f62d9.png">

Why it is?

**Comment by TouvronHugo** [comment]
Hi @mingqiJ ,

For distillation you have to use a [DistilledVisionTransformer](https://github.com/facebookresearch/deit/blob/8ab59156f435cc7be62791ed2edbc65553fc665f/models.py#L20). `deit_small_patch16_224` is not a `DistilledVisionTransformer`. Try with `deit_small_distilled_patch16_224`.
Best,
Hugo
